### PR TITLE
Split surveys into individual folders

### PR DESCRIPTION
**Pull request by rmccar**
### What is the context of this PR?
Currently some surveys such as sand and gravel (survey_ids 066 and 076) and roofing tiles and slate (survey ids 068 and 071) share the same folders in our definitions repo. This makes it more difficult for the SDS team to automate the rollout of SDS schemas when a new version is released. This PR splits them out to fix this issue.

### Links
https://jira.ons.gov.uk/browse/ECI-1400

### Checklist

* [ ] Changes to the spec have been added to example schemas in [examples/](/examples/)
* [ ] JSON Schema definitions have been updated

### CODE REVIEW

**Review by Jon-Kent — APPROVED** [review]

File: examples/sand_and_gravel_marine_dredged/v2.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "schema_version": "v2",
+  "survey_id": "066",
```
Comment (VirajP1002) [inline examples/sand_and_gravel_marine_dredged/v2.json:3]: Something minor, but should the survey_id here be "076"?

File: examples/slate/v2.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
+  "schema_version": "v2",
+  "survey_id": "068",
```
Comment (VirajP1002) [inline examples/slate/v2.json:3]: And should this be "071"?

Comment (rmccar) [reply examples/slate/v2.json]: Good spot, fixed

File: examples/sand_and_gravel_marine_dredged/v2.json (hunk 1 above)
Comment (rmccar) [reply examples/sand_and_gravel_marine_dredged/v2.json]: Fixed

**Review by JacobHarrisonONS — APPROVED** [review]
Looks good. Thanks.

**Review by farres1 — APPROVED** [review]

**Pull request merged**